tests/functional/ext/typing/redundant_union_assign_typehint.rc

```diff
@@ -0,0 +1,6 @@
+[main]
+py-version=3.10
```

Comment:
In my opinion we should separate code that can run in python 3.7 (``Union[int, int]``) from code that need python 3.8 with ``__future__``annotation and python 3.10 specific code. 3.7 is going to be used for a long time. In order to do that we can have multiple file with multiple py-version conf.

Reply:
you mean like having 2 different test names each with its own `rc` file?

Reply:
Yes with one suffixed ``_py310`` with only python 3.10 specific checks, one suffixed ``_py38``... etc.

Reply:
Done. I like the new version of tests, I removed the `py_version` from the `rc` file because I saw that the fake version made python 3.9 tests pass even though they should have failed (before the tests were fixed). I figured all the `#pylint disable` is not needed anymore too :)